grove/connectors/stripe/events.py

```diff
@@ -0,0 +1,58 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: MPL-2.0
+
+"""Stripe Events connector for Grove."""
+
+from stripe import StripeClient
+
+from grove.connectors import BaseConnector
+from grove.constants import REVERSE_CHRONOLOGICAL
+from grove.exceptions import NotFoundException
+
+
+class Connector(BaseConnector):
+    NAME = "stripe_events"
+    POINTER_PATH = "id"
+    LOG_ORDER = REVERSE_CHRONOLOGICAL
+
+    def collect(self):
+        """Collects all events from the Stripe Events API.
+
+        Stripe's list API methods use cursor-based pagination with a unique ID value
+        for each event. The Stripe API allows up to 100 read operations per second,
```

Comment:
Per second, or per request?

Reply:
@hcpadkins Stripe's API [rate limiting page](https://docs.stripe.com/rate-limits#:~:text=We%20have%20several%20limiters%20in,for%20each%20in%20test%20mode.) states 
> For most APIs, Stripe allows up to 100 read operations per second and 100 write operations per second in live mode, and 25 operations per second for each in test mode.

Reply:
There's no constraint, if you have a bunch of Grove connectors configured it's possible this could be exceeded - especially if this is a "global" per-account rate limit rather than per token. That said, if that does occur, Grove will catch the exception, report it, and try again later.